osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyNewStyleSpinner.cs

```diff
@@ -31,12 +31,13 @@ public class LegacyNewStyleSpinner : LegacySpinner
 
         private Container scaleContainer;
 
+        public override Drawable ApproachCircle { get; protected set; }
+
         [BackgroundDependencyLoader]
         private void load(ISkinSource source)
         {
             AddInternal(scaleContainer = new Container
             {
-                Scale = new Vector2(SPRITE_SCALE),
```

Comment:
???

Reply:
Since the transform `LegacySpinner`-side uses `SPRITE_SCALE * 0.1` as the end value, this has to be done. Alternatively, I could move the transform to both the `LegacyNew` and `LegacyOld` and not use the `SPRITE_SCALE` in the transform here.

Reply:
Or use add internal or anything else? You've moved the "scale" way from the "scaleContainer". Without removing the "container" part.

Reply:
Wait actually this shouldn't add the approach circle to the `scaleContainer` as it shouldn't be scaled with the rest, not sure what I was thinking...

Reply:
Have applied the change for this.